aea/context/base.py

```diff
@@ -66,6 +68,11 @@ def public_keys(self) -> Dict[str, str]:
         """Get public keys."""
         return self._public_keys
 
+    @property
+    def addresses(self) -> Dict[str, str]:
+        """Get addresses."""
+        return self._addresses
```

Comment:
let's similarly to below (`public_key`), create also an `address` getter which uses the one associated with the default crypto.